### PR TITLE
SPM Package File

### PR DESCRIPTION
**Pull request by litso**
Add package file to support SPM.
Also update Cocoapods and bump deployment targets.

### CODE REVIEW

File: Sources/Sections/Classes/Sections.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 //  Created by Robert Manson on 7/7/16.
 //
 
-import Foundation
+import UIKit
```
Comment (eerie444) [inline Sources/Sections/Classes/Sections.swift:8]: Is this necessary? [`IndexPath`](https://developer.apple.com/documentation/foundation/indexpath) is part of Foundation. Is there some other UIKit reference in here that I'm missing?

**Review by eerie444 — APPROVED** [review]
I left just one question, otherwise looks good. I verified that adding adding Sections to a project via SPM and CocoaPods both work as expected.

File: Sources/Sections/Classes/Sections.swift (hunk 1 above)
Comment (litso) [reply Sources/Sections/Classes/Sections.swift:8]: Apparently so. If I go to definition it takes me to the generated header file for UIKit. Apparently the convenience methods related to section and row [are defined in NSIndexPath](https://developer.apple.com/documentation/foundation/nsindexpath/1614934-init)

Comment (eerie444) [reply Sources/Sections/Classes/Sections.swift:8]: Fascinating. I wonder who’s out there using that type for anything other than table views and collection views.

Comment (litso) [reply Sources/Sections/Classes/Sections.swift:8]: Yes very strange

**Pull request merged**